src/app/learn/reference/template-items/index.html

```diff
@@ -35,7 +35,7 @@
     </li>
     <li>
       <p>
-        <stache-code>coverage</stache-code> folder — Stores code coverage results for unit tests. This folder is generated when you run <stache-code>skyux test</stache-code> or <stache-code>skyux watch</stache-code>, and it provides a convenient way for developers to see what code the tests reached and what code they did not reach. We recommend that all new code should have 100 percent unit test code coverage. For information about the code coverage requirements to contribute to SKY UX, see the <a href="https://github.com/blackbaud/skyux2/blob/master/CONTRIBUTING.md">SKY UX contribution guidelines</a>.
+        <stache-code>coverage</stache-code> folder — Stores code coverage results for unit tests. This folder is generated when you run <stache-code>skyux test</stache-code> or <stache-code>skyux watch</stache-code>, and it provides a convenient way for developers to see what code the tests reached and what code they did not reach. We recommend that all new code should have 100 percent unit test code coverage. For information about the code coverage requirements to contribute to SKY UX, see the <a routerLink="/contribute/guidelines">SKY UX contribution guidelines</a>.
       </p>
```

Comment:
Should we remove the entry for the `e2e` folder? I thought you asked in Slack and that Alex confirmed it's not created by the template anymore.